### PR TITLE
New RPD and RFD derivative sprites

### PR DESCRIPTION
**Pull request by alsoandanswer**
also adds inhands for them
dreammaker_3sJcg72uaX

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

File: html/changelogs/wezzy_new_rfd_sprites.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Added a changelog editing system that should cause fewer conflicts and more accurate timestamps."
+  - rscdel: "Killed innocent kittens."
```
Comment (NonQueueingMatt) [inline html/changelogs/wezzy_new_rfd_sprites.yml]: bruh

Comment (alsoandanswer) [reply html/changelogs/wezzy_new_rfd_sprites.yml]: FUCK

**Comment by alsoandanswer** [comment]
!review

**Comment by alsoandanswer** [comment]
Merge conflict fixed and tested, should work fine.

**Comment by Geevies** [comment]
Did you make these sprites yourself?

**Comment by alsoandanswer** [comment]
Changed the changelog, thanks for reminding me, Geeves.

**Comment by alsoandanswer** [comment]
!review

**Pull request merged**